### PR TITLE
docker: Automatic starting of db

### PR DESCRIPTION
**Pull request by marusak**
If run for the first time (docker run), then a new DB is created.
If starting stopped container, then start DB and check for new
migrations.

Signed-off-by: Matej Marusak <mmarusak@redhat.com>

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -33,4 +33,4 @@ VOLUME /var/lib/postgres
 VOLUME /var/spool/faf
 
 EXPOSE 80
-CMD ["sh", "-c", "crond && apachectl -D FOREGROUND"]
+CMD ["sh", "-c", "crond && init_db && apachectl -D FOREGROUND"]
```
Comment (mkutlak) [inline docker/Dockerfile]: Wouldn't it be better to create just one init script and include all these into it?

Comment (marusak) [reply docker/Dockerfile]: Great idea! See my new commit in this PR.

File: docker/files/run_faf [hunk 2]
```diff
@@ -27,3 +27,5 @@ else
     sudo -u postgres /usr/bin/pg_ctl start -D /var/lib/postgres/data -w > /dev/null
     sudo -u faf faf-migrate-db
 fi
+crond
+init_db && apachectl -D FOREGROUND
```
Comment (mkutlak) [inline docker/files/run_faf]: There is no init_db anymore, and "-D FOREGROUND" should be "-DFOREGROUND".

Comment (marusak) [reply docker/files/run_faf]: Yeah, I forgot to push the fix with init_db. Fixed. And are you sure about the `-DFOREGROUND`? I think that separated works as well.

Comment (mkutlak) [reply docker/files/run_faf]: Yes, it works, but you get "Passing arguments to httpd using apachectl is no longer supported." warning. -DFOREGROUND is special parameter, check man page of httpd.

**Comment by mkutlak** [comment]
LGTM. Thanks!

**Pull request merged**